artemis-console-extension/artemis-extension/public/artemisconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "jmx": {
+      "domain": "org.apache.activemq.artemis"
+    }
+}
```

Comment:
Is there some validation done in the pipelline at some point? I'm thinking about a jsonschema of some sort.

Reply:
> Is there some validation done in the pipelline at some point? I'm thinking about a jsonschema of some sort.

There isn't, tbh I took this code from hawtIO, it is how they do it with hawtconfig.json. I wouldnt know how to do whatyou suggest tbh.

Reply:
That's alright, that was mainly an out-of-curiosity question. No reason to diverge from hawtio in any case.